templates/data/mongodb/managed.html

```diff
@@ -0,0 +1,466 @@
+                <br />
+                per node*
+              </p>
+            </div>
+          </div>
+          <p>Secure, managed service delivered to your on-premise data centre by our MongDB operations team.</p>
+          <hr class="is-muted" />
+          <div class="p-section--shallow u-no-margin--bottom">
+            <ul class="p-list--divided u-no-margin--bottom">
+              <li class="p-list__item is-ticked">
+                Deliver a web-scale MongoDB data platform on-premise (bare metal) with our managed services team
+              </li>
+              <li class="p-list__item is-ticked">Maximize efficiency gains</li>
+              <li class="p-list__item is-ticked">SLA and compliance commitments so data isn't compromised</li>
+              <hr class="is-muted" />
+            </ul>
+            <p class="u-text--muted">* All nodes in the cluster must be covered. Per-VM pricing is also available.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <hr class="is-fixed-width" />
+
+  <section class="p-strip is-deep">
+    <div class="u-fixed-width">
+      <p class="p-heading--2">
+        <a href="https://ubuntu.com/managed">Learn more about Managed Services&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>MongDB resources</h2>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <h3 class="p-heading--5 u-no-margin--bottom">
+            <a href="https://ubuntu.com/engage/mongodb-for-enterprises">Webinar: MongoDB for modern data management</a>
+          </h3>
+          <p>Hear enterprise use cases and how MongoDB's key features are yielding benefits for modern data management.</p>
+        </div>
+        <hr class="is-muted" />
+        <div class="p-section--shallow">
+          <h3 class="p-heading--5 u-no-margin--bottom">
+            <a href="https://ubuntu.com/engage/simplifying-mongodb-operations">Webinar: Simplifying MongoDB operations</a>
+          </h3>
+          <p>Learn how to navigate the complexity of managing your database with Charmed MongoDB.</p>
+        </div>
+        <hr class="is-muted" />
+        <div class="p-section--shallow">
+          <h3 class="p-heading--5 u-no-margin--bottom">
+            <a href="https://ubuntu.com/engage/mongodb-support-security">Whitepaper: MongoDB<sup>&reg;</sup> Toolkit</a>
+          </h3>
+          <p>A guide to streamlining MongoDB database operations including security and support.</p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="p-section--deep">
+    <div class="u-fixed-width">
+      <hr />
+      <p class="u-text--muted">
+        "MongoDB" is a trademark or registered trademark of MongoDB, Inc. Other trademarks are property of their respective owners. Charmed MongoDB is not sponsored, endorsed, or affiliated with MongoDB, Inc.
```

Comment:
This should be moved to the right
<img width="582" alt="Screenshot 2024-08-22 at 1 54 11 PM" src="https://github.com/user-attachments/assets/f9d44c25-03f3-4489-bf14-8ec749277e13">